.github/workflows/release-package.yml

```diff
@@ -0,0 +1,33 @@
+# Publish a new package when a pre-release PR is merged.
+#
+# Job does nothing if PR that was merged is not a pre-release PR.
+
+name: "Publish @webref package if needed"
+
+on:
+  pull_requests:
+    branches:
+      - 'release-*'
+    types: [closed]
```

Comment:
Will this trigger is the PR is closed without merging too?

Reply:
Yes, hence the need for the `if: github.event.pull_request.merged == true` check 4 lines below. I could not find a direct way to trigger a worfklow on a merge action.